Makefile

```diff
@@ -3,19 +3,26 @@ TEST_LDLIBS = -lgtest
 SRC_DIR = src
 TEST_DIR = tests
 BUILD_DIR = build
+RFL_DIR=/usr/local/include/rfl
```

Comment:
Please add a `TODO` about creating a doc describing how to compile this project including the libraries to download and install